src/login.html

```diff
@@ -18,35 +20,40 @@
             <div class="row g-0">
               <div class="col-lg-6">
                 <div class="card-body p-md-5 mx-md-4">
-  
+
                   <div class="text-center">
-                    <img src="img/company-logo.svg"
-                      style="width: 185px;" alt="logo">
-                      <br></br>
-                    <h4 class="mt-1 mb-5 pb-1">Embedded Analytics Portal</h4>
+                    <img src="img/company-logo.svg" style="width: 185px;" alt="logo">
+                    <br></br>
+                    <h4 class="mt-1 mb-5 pb-1">My Company's App</h4>
                   </div>
-  
+
                   <form action="/login" method="post">
                     <p>Please login to your account</p>
-  
+
                     <div class="form-outline mb-4">
-                      <input type="email" id="email" name="email" class="form-control" placeholder="email address" value="" required />
+                      <input type="email" id="email" name="email" class="form-control" placeholder="email address"
+                        value="" required />
                       <br>
```

Comment:
- Add a label for accessibility.
- Remove the `<br>` tag and handle spacing with the `mt-3` class.
```suggestion
                      <label for="password" class="mt-3">Password</label>
```